### DISCUSSION TITLE
How do you Create Collection?

### DISCUSSION THREAD

**Discussion by Agwebberley** [jrief/django-formset#71] [Q&A]
There is EditCollectionView, and I have looked all through the docs and I can't find anywhere where it explains how to create a collection.  I am trying to make a simple ordering system, where you have:

- Order
  - Customer (Foreign Key)
  - Order Items
    - Item (Foreign Key)
    - Quantity

All created in the same form (Items and customer are created in another form but selected in this form)

**Comment by jrief** [comment]
> There is EditCollectionView, and I have looked all through the docs and I can't find anywhere where it explains how to create a collection

It's documented here: https://django-formset.fly.dev/model-collections/
